bitcoin_tools/analysis/status/utils.py

```diff
@@ -248,13 +255,14 @@ def decode_utxo(coin, outpoint, version=0.15):
 
         # And the remaining data corresponds to the script.
         script = coin[offset:]
+        decompressed_script = decompress_script(out_type,script)
```

Comment:
`decode_utxo` is one of the most time consuming functions of the library (not by itself, but when used to decode all the outputs from the `chainstate`). Adding additional computation to it it's definitely not recommended. If you want to get the full hex script for every output it's better to do it with the return of the function, since it is not always necessary.